packages/fx-core/resource/package.nls.json

```diff
@@ -745,7 +745,7 @@
   "error.yaml.InvalidActionInputError": "The '%s' action cannot be completed as the following parameter(s): %s, are either missing or have an invalid value in the provided yaml file: %s. Ensure that the required parameters are provided and have valid values and try again.",
   "error.common.InstallSoftwareError": "Unable to install %s. You can install it manually and restart Visual Studio Code if you are using the Toolkit in Visual Studio Code.",
   "error.common.VersionError": "Unable to find a version satisfying the version range %s.",
-  "error.common.MissingEnvironmentVariablesError": "The program cannot proceed as the following environment variables are missing: '%s', which are required for file: %s. Make sure the required variables are set either by editing the .env file '%s' with the correct names and values , or by setting the system environment variables with the correct names and values. If you are developing with a new project created with Teams Toolkit, running provision or debug will register correct values for these environment variables.",
+  "error.common.MissingEnvironmentVariablesError": "The program cannot proceed as the following environment variables are missing: '%s', which are required for file: %s. Make sure the required variables are set either by editing the .env file '%s' or '%s' with the correct names and values, or by setting the system environment variables with the correct names and values. If you are developing with a new project created with Teams Toolkit, running provision or debug will register correct values for these environment variables.",
```

Comment:
Proposed: The program cannot proceed because the following environment variables are missing: '%s' for file: %s. Please set them by editing the .env file '%s' or '%s', or the system environment variables with the correct names and values. For new Teams Toolkit projects, running provision or debug will register correct values for these environment variables.